app/scripts/components/exploration/data-utils-no-faux-module.ts

```diff
@@ -139,6 +139,18 @@ const hasValidSourceParams = (params) => {
   return params && 'colormap_name' in params && 'rescale' in params;
 };
 
+/**
+ * Utility to check if render parameters are applicable based on dataset type.
+ *
+ * @param datasetType The type of the dataset (e.g., 'vector').
+ * @returns Boolean indicating if render parameters are applicable.
+ */
+export const isRenderParamsApplicable = (datasetType: string): boolean => {
```

Comment:
can we possibly type this as `DatasetLayerType`

Reply:
Thanks for spotting, updated!